src/permutat.c

```diff
@@ -4443,6 +4443,7 @@ Obj FuncMappingPermListList(Obj self, Obj src, Obj dst)
     Obj out;
     Obj tabdst, tabsrc;
     Int x;
+    Obj obj;
     Int mytabs[DEGREELIMITONSTACK+1];
     Int mytabd[DEGREELIMITONSTACK+1];
 
```

Comment:
Right, but not a very meaningful error, in this context, it'd be easier to find the source of the problem if there is an explicit error here.